afterburners/client.lua

```diff
@@ -26,52 +23,46 @@ else
 end
 	
 
-Citizen.CreateThread(function()
+CreateThread(function()
 	while true do
-		Citizen.Wait(50)
```

Comment:
again, why?

Reply:
This is the exact same as Wait, as in 
`
Citizen.Wait = Wait
`
So in theory is faster

Reply:
see above

Reply:
See above